libcodechecker/analyze/analyzer.py

```diff
@@ -178,6 +178,35 @@ def perform_analysis(args, context, actions, metadata):
         statistics_data = manager.dict({'stats_out_dir':
                                         args.stats_output})
 
+    if 'stats_min_sample_count' in args:
```

Comment:
If I remember it correctly the default value will be always set so no checking is needed.